app-component.html

```diff
@@ -35,13 +35,11 @@ <h2 class="headline">Introduction</h2>
       </div>
     </div>
 
-    <hr ng-if="::!$ctrl.config.productionMode" />
+    <hr ng-if="::!$ctrl.config.productionMode">
     <br-demo-warning></br-demo-warning>
 
-    <div ng-if="$ctrl.config.footer.show">
-      <hr/>
-      <br-footer ng-if="$ctrl.config.footer.show"></br-footer>
-    </div>
+    <hr ng-if="!$ctrl.config.footer._hideOuterHr">
+    <br-footer ng-if="$ctrl.config.footer.show"></br-footer>
```

Comment:
Can we do `ng-if="$ctrl.config.footer.show || $ctrl.route.vars.footer.show"` and start to transition away from having the config in the scope?